crates/cast/src/lib.rs

```diff
@@ -128,8 +128,10 @@ where
         req: &WithOtherFields<TransactionRequest>,
         func: Option<&Function>,
         block: Option<BlockId>,
+        json: Option<bool>,
```

Comment:
Likely able to simplify by just passing `bool` directly making unwrapping unnecessary